packages/website/src/css/custom.css

```
@@ -71,81 +71,44 @@ html[data-theme='dark']:root {
   --gray-border-shadow: var(--ifm-color-secondary-dark);
 }
 
-.bug-report-link {
+.image-link {
   display: flex;
   flex-direction: row;
   align-items: center;
-  justify-content: start;
+  justify-content: flex-start;
 }
-.bug-report-link:before {
+
+.image-link:before {
   content: '';
   display: flex;
+}
+
+/* assumes icons are black by default and inverts them for dark mode so they're white */
+html[data-theme='dark'] .image-link:before {
+  filter: invert(100%);
+}
+
+.bug-report-link:before {
   background: url('/img/bug-report.svg') no-repeat;
 }
-.github-link {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: start;
+.discord-link:before {
+  background: url('/img/discord.svg') no-repeat;
```

```suggestion
  background: url('/img/discord.svg') no-repeat;
  background-position: center;
```

this is a nit, as this icon is not squared, we have to align it